src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/entity/BlockEntity.kt

```diff
@@ -0,0 +1,26 @@
+package com.chattriggers.ctjs.minecraft.wrappers.entity
+
+import com.chattriggers.ctjs.minecraft.wrappers.world.block.BlockPos
+import com.chattriggers.ctjs.utils.MCBlockEntity
+
+// TODO(breaking): Renamed from TileEntity to BlockEntity
+class BlockEntity(val blockEntity: MCBlockEntity) {
+    fun getX(): Int = getBlockPos().x
+
+    fun getY(): Int = getBlockPos().y
+
+    fun getZ(): Int = getBlockPos().z
+
+    // TODO
+//    fun getBlock(): Block = Block(getBlockType(), getBlockPos())
+
+//    fun getBlockType(): BlockType = BlockType(blockEntity.type)
```

Comment:
Honestly look at `BlockEntityType`, it doesn't seem like it really has any useful information, so we might want to just remove these.